src/Bridges/Constraint/bridges/quad_to_soc.jl

```diff
@@ -226,6 +244,35 @@ function MOI.get(
     return output
 end
 
+function MOI.set(
+    model::MOI.ModelLike,
+    attr::MOI.ConstraintPrimalStart,
+    bridge::QuadtoSOCBridge{T},
+    value,
+) where {T}
+    soc = MOI.get(model, MOI.ConstraintFunction(), bridge.soc)
+    s = MOI.Utilities.eval_variables(soc) do v
+        # If not set, we assume x = 0.
```

Comment:
Not sure about setting zero here. If the user sets the `VariablePrimalStart` after `ConstraintPrimalStart`, I prefer an error than incorrect result that would be a nightmare to debug. If it's just for making tests pass, let's just add something to the tests

Reply:
> I prefer an error

Shouldn't the start values silently skip if unsupported? Also, I don't know if we have any precedence that says variable primal must be set before constraint primal.

Reply:
This is also a kinda pointless exercise. Because setting the constraint primal for the quadratic does not give enough information to set the constraint primal for the RotatedSecondOrderCone.

And if VariablePrimalStart is not set, then isn't x = 0 a reasonable starting point?

Reply:
Maybe this is another case for https://github.com/jump-dev/JuMP.jl/pull/3271#issuecomment-1458711074

Reply:
> Shouldn't the start values silently skip if unsupported?

If the solver does not support it yes but here it does, it's just the user did not set it correctly so an error is expected so that users fixes his code.

If the `VariablePrimalStart` is not set, I don't think that it's so useful to have this work. If we assume it to be zero, we could as well have the user set it to zero explicitly. At least he's making this sloppy choice.

It didn't work before so having it work in the case variable starting points are set before is already an improvement. If we get complains of users wanting to set it without setting as variable starting value of setting the variable starting value after, we could try to fix that case but I prefer not silently doing weird like having a zero variable starting value.

Reply:
If the user sets the constraint start first and then the variable start, the result he will get is completely wrong and will be so awful to debug. Compared to that, an error message is so much better